apps/condo/domains/acquiring/schema/GeneratePaymentLinkService.js

```diff
@@ -0,0 +1,270 @@
+/**
+ * Generated by `createservice acquiring.GeneratePaymentLinkService`
+ */
+const conf = require('@condo/config')
```

Comment:
best to reorder imports:

Imports from other libs

Imports from condo common modules or packages

Imports from condo local modules

Local imports

Reply:
ok